src/data_factory_testing_framework/_expression_runtime/data_factory_expression/expression_transformer.py

```diff
@@ -86,15 +79,15 @@ def __init__(self) -> None:
             EXPRESSION_LOGICAL_BOOL: "or" | "and"
             EXPRESSION_NULL: NULL
             EXPRESSION_PARAMETER_NAME: /[a-zA-Z0-9_]+/
-            EXPRESSION_PIPELINE_PROPERTY.2: "parameters" | "globalParameters"
+            EXPRESSION_PIPELINE_PROPERTY: /[a-zA-Z0-9_]+/
```

Comment:
What's the reason of dropping the priority?

Reply:
I see now, you merge any pipeline().<parameter> into a single rule and leverage the logic apps runtime to approach pipeline().parameters.<name> and pipeline().globalParamters.<name> just like any other pipeline().<name>